packages/shared/src/components/checklist/OnboardingChecklistBar.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { ReactElement, useContext, useEffect, useState } from 'react';
```

Comment:
Why are this a ChecklistBar 2 different components? I see and understand the need for separation, but the namings though confuse me on what could be on one and what on the other.

Reply:
Basically `ChecklistCard`, `ChecklistBar` and `useChecklist` are like primitives that you use to build actual checklists in the app. If that makes sense?